mPower2/mPower2/View Controllers/TrackingViewController.swift

```diff
@@ -0,0 +1,156 @@
+        let taskGroups: [RSDTaskGroup] = {
+            let activeTaskGroup : RSDTaskGroup = {
+                let taskInfos = MCTTaskIdentifier.all().map { MCTTaskInfo($0) }
+                var taskGroup = RSDTaskGroupObject(with: "Measuring", tasks: taskInfos)
+                taskGroup.title = "Measuring"
+                return taskGroup
+            }()
+            let trackingTaskGroup : RSDTaskGroup = {
+                var taskInfo = RSDTaskInfoObject(with: "Triggers")
+                taskInfo.title = "Triggers"
+                if let image = try? RSDImageWrapper(imageName: "TriggerIcon-oval") {
+                    taskInfo.icon = image
+                }
+                taskInfo.resourceTransformer = RSDResourceTransformerObject(resourceName: "Triggers")
+                var taskGroup = RSDTaskGroupObject(with: "Tracking", tasks: [taskInfo])
+                taskGroup.title = "Tracking"
+                return taskGroup
+            }()
+            
+            return [trackingTaskGroup, activeTaskGroup]
+        }()
+        return taskGroups
+    }
+    
+    func setupView() {
+        
+        // Add our task browser view controller
+        let browser = TaskBrowserViewController()
+        browser.taskGroups = taskGroups()
+        browser.view.translatesAutoresizingMaskIntoConstraints = false
+        addChildViewController(browser)
+        taskBrowserContainer.addSubview(browser.view)
+        browser.view.rsd_alignAllToSuperview(padding: 0.0)
+        
+        // Configure actionBarView
+        actionBarView.layer.cornerRadius = 4.0
+        actionBarView.layer.masksToBounds = true
+        
+        // TODO: jbruhin 5-1-18 update 'welcome' text dynamically
+    }
+    
+    func updateActionBar() {
+        // TODO: jbruhin 5-1-18 will be a data source for this at some point
+        actionBarTitleLabel.text = "Study Burst"
+        actionBarDetailsLabel.text = "4 ACTIVITIES TO DO"
+        
+        // TODO: jbruhin 5-1-18 will need to make the following dynamic based on...something
+        actionBarView.isHidden = !shouldShowActionBar
+        if shouldShowActionBar {
+            // Add some top inset on the tableView
+            let inset = actionBarView.frame.origin.y + actionBarView.frame.size.height
+            tableView.contentInset = UIEdgeInsets(top: inset, left: 0.0, bottom: 0.0, right: 0.0)
+        }
+        //
+    }
+    
+    func updateProgressCircle() {
+        // TODO: jbruhin 5-1-18 will be a data source for this at some point
+        progressCircleView.displayDay(count: 14)
+    }
+    
+    // MARK: Actions
+    @IBAction func actionBarTapped(_ sender: Any) {
+        // TODO: jbruhin 5-1-18 implement
+    }
+}
+
+extension TrackingViewController: RSDTaskViewControllerDelegate {
+
+    // MARK: RSTTaskViewControllerDelegate
+    func taskController(_ taskController: RSDTaskController, didFinishWith reason: RSDTaskFinishReason, error: Error?) {
```

Comment:
Is tapping on these elements suppose to bring up a new instance of the task? I figured they were just visual elements and not suppose to be interactive.

Reply:
I'm pretty sure they're supposed to be tappable. I'll ask Woody

Reply:
I'm confused. Sorry.

What is this vs. TaskBrowserViewController? I thought that was your tab view? It also has an implementation for the task delegate.

Reply:
RSTTaskVC delegate methods removed from this class